src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: auto;
+  margin-right: 4px;
+  display: block;
+}
+
+.stars--4 .stars__star {
+  background-image: url(./../images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(n + 5) {
```

Comment:
implement this requirement 
![image](https://github.com/mate-academy/layout_product-cards/assets/70009205/857d4bee-fc0b-4be4-8032-4478c6682dd3)
![image](https://github.com/mate-academy/layout_product-cards/assets/70009205/2456a6a7-7bad-496c-ac6a-9b4b4930fe99)

Reply:
done:)